### ISSUE TITLE
ValueError: cannot convert float NaN to integer

### ISSUE THREAD

**Issue by fepegar** [thenineteen/Semiology-Visualisation-Tool#136]
When querying "awesome", neutral, neutral in Slicer.

```python
RuntimeWarning: invalid value encountered in double_scalars
  normalizedScore /= (maxScore - minScore)
Traceback (most recent call last):
  File "/Users/fernando/git/Semiology-Visualisation-Tool/slicer/SemiologyVisualisation.py", line 666, in updateColors
    min2dOpacity=self.min2dOpacitySlider.value,
  File "/Users/fernando/git/Semiology-Visualisation-Tool/slicer/SemiologyVisualisation.py", line 1262, in setScoresColors
    color = self.getColorFromScore(normalizedScore, colorNode)
  File "/Users/fernando/git/Semiology-Visualisation-Tool/slicer/SemiologyVisualisation.py", line 1283, in getColorFromScore
    scoreIndex = int(round((numColors - 1) * normalizedScore))
ValueError: cannot convert float NaN to integer
```

**Comment by fepegar** [comment]
This happens because the score is the same for all regions and therefore `maxScore - minScore` is 0. The result should just show all those regions with the brightest colour.

```python
{103: 1.0, 104: 1.0, 173: 1.0, 174: 1.0}
```

**Comment by fepegar** [comment]
Fixed in fc613d06c5f12f9de3f8fd2b72759e723ebe92b0.

**Comment by thenineteen** [comment]
Thanks
